modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -1,3 +1,4 @@
+[[inline-formatting]]
```

Comment:
This didn't have an ID so my IDE complained as anything referenced really should have an explicit id to prevent the link from breaking later on.